.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,108 @@
+name: "🐛 Bug Report"
+description: "If something isn't working as expected 🤔."
+title: "[Bug]: "
+labels: ["i: needs triage"]
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for taking the time to file a bug report! Please fill out this form as completely as possible.
+
+  - type: checkboxes
+    attributes:
+      label: ""
+      options:
+        - label: Would you like to work on a fix?
+
+  - type: dropdown
+    attributes:
+      label: How are you using Babel?
+      options:
+        - babel-loader (webpack)
+        - "@rollup/plugin-babel"
+        - "@babel/eslint-parser"
+        - "@babel/cli"
+        - "@babel/register or @babel/node"
+        - Programmatic API (`babel.transform`, `babel.parse`)
+        - Other
```

Comment:
should we put like Other (or not sure)? or reference a framework somewhere (next.js etc)?

Reply:
```suggestion
        - Other (Next.js, Gatsby, vue-cli, ...)
```